css/01-layout.css

```diff
@@ -186,22 +186,37 @@ footer.site-footer {
 	padding: 4rem 0 !important;
 }
 
+.article-body.side-right .one {
+	padding-right: 2rem !important;
```

Comment:
🙈

Reply:
👍  !important !!!

Reply:
I will make a note to look for another solution to the `!important` tags in a future push.

Reply:
`#jacket #binder` all the way!